src/main/java/fr/inria/gforge/spoon/SpoonMojoGenerate.java

```diff
@@ -136,6 +139,17 @@
 	)
 	private boolean skipSpoonErrors;
 
+	/**
```

Comment:
why is this change required? where is it used?

Reply:
it gives you the option to split testFiles and srcFiles in handling, if i understood the code correct. Currently it's not used, but allows future extensions easier or?

Reply:
If it's not used, then let's remove it

Reply:
removed